alchemist-incarnation-biochemistry/src/test/resources/provaBCReaction.yml

```diff
@@ -17,7 +17,9 @@ deployments:
   - type: Circle
     parameters: [20, 0, 0, 5]
     nodes:
-      type: CellNodeImpl
-      parameters: [1, p] # p useless parameter
+      type: GenericNode
```

Comment:
not needed

Reply:
This is actually needed. The biochemistry incarnation load the node with a Cellular property since we are prohibiting adding 2 node property of the same type, without the node type specification an exception will be thrown.